.sample_configs/process_configs.yaml

```diff
@@ -182,6 +183,7 @@ get_model_evaluation_sample:
   - model_explanation
 get_model_evaluation_slice_sample: {}
 get_model_evaluation_tabular_classification_sample: {}
+get_model_evaluation_tabular_forecasting_sample: {}
```

Comment:
If the `tabular_forcasting` variant for `get_model_evaluation` ends up identical with any `get_model_evaluation`, please skip it.  (and we should separately clean up the others - but perhaps i missed something here)

Reply:
I got it, we want to consolidate if possible. Let me do another pass.